pyresample/test/test_geometry/test_area.py

```diff
@@ -25,63 +25,279 @@
 from pyproj import CRS, Proj
 
 import pyresample
+import pyresample.geometry
 from pyresample import geo_filter, parse_area_file
 from pyresample.future.geometry import AreaDefinition, SwathDefinition
 from pyresample.future.geometry.area import (
+    _get_geostationary_bounding_box,
     get_full_geostationary_bounding_box_in_proj_coords,
     get_geostationary_angle_extent,
-    get_geostationary_bounding_box_in_lonlats,
-    get_geostationary_bounding_box_in_proj_coords,
     ignore_pyproj_proj_warnings,
 )
 from pyresample.future.geometry.base import get_array_hashable
 from pyresample.geometry import AreaDefinition as LegacyAreaDefinition
 from pyresample.test.utils import assert_future_geometry
 
 
+# BUG in 'area_class' fixture
+# --> Overwrite create_test_area here
+def create_test_area(crs, shape, area_extent, **kwargs):
+    """Create an AreaDefinition object for testing."""
+    area = AreaDefinition(crs=crs, shape=shape, area_extent=area_extent, **kwargs)
+    return area
+
```

Comment:
Seems that `pytest` fixture has a bug with nested call when passed to `pytest` mark parametrize. 
Instead of using the fixture defined in `conftest`, this enable to test stuffs. Maybe @djhoese you want to check it out ... I spend 3-4 hours ... and did find a way to fix it